python/Ganga/Runtime/bootstrap.py

```diff
@@ -1279,7 +1279,8 @@ def launch_IPython(local_ns, args, error_handler, ganga_prompt):
         ipshell.confirm_exit = config['confirm_exit']
 
         # Launch embedded shell
-        ipshell(local_ns=local_ns)
+        from Ganga import GPI
+        ipshell(local_ns=local_ns, module=Ganga.GPI)
```

Comment:
Shouldn't the ```local_ns``` then also be ```Ganga.GPI``` if we're worrying about polluting the interactive command line?
Historically the ```local_ns``` as it is now was mapped to both the ```local``` and ```global``` yes, I was never sure about this but copied what was being done with a much older version of IPython.